client/graphics/animation.hpp

```diff
@@ -8,7 +8,7 @@
 #include <glm/glm.hpp>
 #include <glm/ext.hpp>
 
-#include "components.pb.h"
+#include <components.pb.h>
```

Comment:
Does this sort out the warnings being generated based on protobuf's code?

Reply:
Sadly on windows nope.